template/page-community.html.jinja

```diff
@@ -99,6 +257,97 @@
 {% endblock %}
 
 
+{% block endbody %}
+<script type="text/javascript" src="{{currentpage.prefix}}assets/js/bodymovin.min.js"></script>
+
+<!-- Light version for network node looks ok for both light/dark.-->
+<!-- Dark version for other two and using filter invert and adjust brightness for light mode -->
+<!-- <script type="text/javascript" src="{{currentpage.prefix}}assets/js/community/network-node-dark.json"></script> -->
+<script type="text/javascript" src="{{currentpage.prefix}}assets/js/community/network-node-light.json"></script>
+<script type="text/javascript" src="{{currentpage.prefix}}assets/js/community/XRPL-grants-dark.json"></script>
+<!-- <script type="text/javascript" src="{{currentpage.prefix}}assets/js/community/XRPL-grants-light.json"></script> -->
+<script type="text/javascript" src="{{currentpage.prefix}}assets/js/community/careers-dark.json"></script>
+<!-- <script type="text/javascript" src="{{currentpage.prefix}}assets/js/community/careers-light.json"></script> -->
```

Comment:
Please clean up the commented-out stuff here too.

Reply:
deleted and removed extra files in assets